StoredBucket.php

```diff
@@ -2,13 +2,23 @@
 
 namespace iFixit\TokenBucket;
 
-use \DateTime;
+use \InvalidArgumentException;
 
 class StoredBucket {
    private $tokens;
    private $lastConsume;
 
-   public function __construct($tokens, DateTime $lastConsume) {
+   /**
+    * @param $lastConsume number of seconds since the last consumption.
+    */
+   public function __construct($tokens, $lastConsume) {
+      if (!is_double($lastConsume)) {
+         throw new InvalidArgumentException("lastConsume should be an int");
+      }
```

Comment:
Yeah, that's a good point to bring up, someone should be able to use either an int or double. I'll change it to is_numeric()